.buildkite/pipeline.yml

```diff
@@ -11,19 +11,17 @@ steps:
       git fetch origin
       buf breaking --against ".git#ref=`git merge-base origin/master HEAD`"
 
-    timeout: 30
     agents:
-    - "distro=amazonlinux"
+      - "queue=gcp-standart"
```

Comment:
Call my crazy but shouldn't it be spelled standar**d**?

Reply:
you're sure not. I used standarD everywhere else except queue names, will change it as well